app/assets/stylesheets/components/_post.scss

```diff
@@ -80,6 +83,64 @@ $Post-inlineCodeBgColor: rgba(0, 0, 0, 0.04);
     }
   }
 
+  .Table {
```

Comment:
I'm fine with having that line 27 break the rules, given the complexity of generating a proper Suit-compliant markup from the markdown table.